tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs

```diff
@@ -1883,6 +1931,11 @@ private void VerifyDelete(int tournamentId, Times times)
             _tournamentServiceMock.Verify(ts => ts.Delete(tournamentId), times);
         }
 
+        private void VerifyActivate(int tournamentId, Times times)
+        {
+            _tournamentServiceMock.Verify(ts => ts.UnArchive(tournamentId), times);
+        }
+
```

Comment:
Why two method that do the same? VerifyUnArchiveTournament and Verify Activate(rename method to VerifyActivateTournament)